src/bin/vip-config-software.js

```diff
@@ -2,10 +2,36 @@
 
 import command from '../lib/cli/command';
 
+const usage = 'vip config software';
+const exampleUsage = 'vip @example-app.develop config software';
+const exampleUsageNode = 'vip @example-node-app.develop config software';
+
+// Command examples
+const examples = [
+	{
+		usage: `${ exampleUsage } get`,
+		description: 'Retrieve the current software configuration for all supported components.',
```

Comment:
Suggested:
`Retrieve a list of the current versions of all environment software in the default table format.`